compat/main.py

```diff
@@ -122,17 +111,17 @@ class GoTest:
     path: str
     command: str
 
-    def run(self, working_dir: Path, prepare: bool, go_test_ref: Optional[str]) -> bool:
-        if go_test_ref:
-            replacement = f'github.com/onflow/cadence@{go_test_ref}'
-        else:
-            replacement = shlex.quote(str(Path.cwd().parent.absolute()))
+    def run(self, working_dir: Path, prepare: bool, cadence_version: str, flowgo_version: str) -> bool:
```

Comment:
Maybe keep the Cadence version optional, and keep the original `else` branch. The idea here is that when not specifying a particular version, the default is to use the working copy of Cadence, by using the absolute path to it. This is useful for local development